peripherals/cordic/cordic_g4.yaml

```diff
@@ -0,0 +1,50 @@
+# CORDIC coprocessor provides hardware acceleration
+# for computing math funcitons
+
+CORDIC:
+  CSR:
+    RRDY:
+      _read:
+        NotReady: [0, "Results from computation are not read"]
+        Ready: [1, "Results are ready, this flag will be automatically cleared once value is read"]
+      _write:
+        Clear: [0, "Clear the computation results"]
```

Comment:
```suggestion
```

From looking at RM0440, it seems like this field is read-only, and is cleared by hardware when you read RDATA enough, rather than being clearable from software?

Reply:
Yes, thanks for catching it! I wrote it while still wrapping my head around the docs and the DSL and haven't checked again.

Just checked in the docs and RRDY is indeed marked as read only.